packages/editor-ui/vite.config.mts

```diff
@@ -74,6 +76,14 @@ const plugins = [
 		compiler: 'vue3',
 		autoInstall: true,
 	}),
+	components({
```

Comment:
could we add this to the design system? so we could use it there while also still being able to use it directly here

for example, how would we use this with `n8n-icon` or `n8n-button`?

Reply:
Added to the design system. I don't have the time to replace the FontAwesome usages with this, so we'll need to leave that for another time.